lambda/schema.ts

```diff
@@ -13,17 +14,9 @@ export const CONTENT_TYPES = {
 
 export const headersSchema = z
   .object({
-    'content-type': z.string().optional(),
-    'Content-Type': z.string().optional()
+    'content-type': z.nativeEnum(CONTENT_TYPES)
   })
-  .refine(
-    headers => {
-      return headers['content-type'] || headers['Content-Type'];
-    },
-    {
-      message: 'Missing Content-Type header'
-    }
-  );
+  .transform(obj => mapKeys(obj, (_, key) => key.toLowerCase()));
```

Comment:
This effectively calls .toLowerCase() on all the header keys prior to validating against the schema. So if GitHub changed the header to CONTENT-TYPE it would still pass